app/build.gradle

```diff
@@ -12,10 +15,10 @@ android {
     }
     signingConfigs {
         release {
-            storeFile file("C:/Users/tejus/Documents/Repos/UnityClient/Assets/AndroidWebView/Dependencies/altspacevr-client.keystore")
-            storePassword "vpiu+qTp9uiXq7J>"
-            keyAlias "altvrclient"
-            keyPassword "vpiu+qTp9uiXq7J>"
+            storeFile file(properties.getProperty("signing.storeFile"))
```

Comment:
Thanks! either way, it should be okay if they didn't create a release  / signed apk for us.